### PR TITLE
Elastic Defend installation/onboarding changes for 8.7

### LINKED ISSUE #2963: Updates to installation/onboarding for Elastic Defend
## Description 

The installation process for Elastic Defend has been streamlined in 8.7. More details to be added to this issue soon, but creating it now as a placeholder.

## Acceptance Test Criteria

Run through the current docs for installation, compare, update to match the current installation in 8.7.

## Notes

- To be added

### PR DESCRIPTION
**Pull request by joepeeples**
Resolves #2963.

Preview: [Add the Elastic Defend integration](https://security-docs_3069.docs-preview.app.elstc.co/guide/en/security/master/install-endpoint.html#add-security-integration)

### CODE REVIEW

**Comment by joepeeples** [comment]
@elasticmachine run elasticsearch-ci/docs

**Comment by joepeeples** [comment]
@elasticmachine run elasticsearch-ci/docs

**Comment by joepeeples** [comment]
@elasticmachine run elasticsearch-ci/docs

**Review by kevinlog — APPROVED** [review]

File: docs/getting-started/install-endpoint.asciidoc [hunk 1]
```diff
@@ -34,37 +34,37 @@ image::images/install-endpoint/endpoint-cloud-sec-integrations-page.png[Search r
...
-. Select a configuration preset. Each preset comes with different default settings for {agent} — you can further customize these later. To learn more, refer to <<configure-endpoint-integration-policy, configure Elastic Defend integration policies>>.
+. Select a configuration preset. Each preset comes with different default settings for {agent} — you can further customize these later by <<configure-endpoint-integration-policy,configuring the {elastic-defend} integration policy>>.
 +
 [cols="2"]
 |===
 |*Traditional Endpoint presets*
-| All traditional endpoint presets have the following preventions enabled by default: machine learning malware, ransomware, memory threat, malicious behavior, and credential theft. Each preset collects the following events:
-
-*Next-Generation Antivirus (NGAV):* Process
+a| All traditional endpoint presets except *Data Collection* have these preventions enabled by default: malware, ransomware, memory threat, malicious behavior, and credential theft. Each preset collects the following events:
```
Comment (nastasha-solomon) [inline docs/getting-started/install-endpoint.asciidoc:49]:
```suggestion
a| All traditional endpoint presets, except *Data Collection*, have these preventions enabled by default: malware, ransomware, memory threat, malicious behavior, and credential theft. Each preset collects the following events:
```

Comment (nastasha-solomon) [reply docs/getting-started/install-endpoint.asciidoc:49]: Italics work! And yes it was to emphasize that Data Collection was excluded.

**Review by benironside — APPROVED** [review]
LGTM!

**Pull request merged**